src/SpacetimeDBClient.cs

```diff
@@ -80,7 +78,8 @@ struct DbOp
         /// <summary>
         /// Invoked when an event message is received or at the end of a transaction update.
         /// </summary>
-        public event Action<ClientApi.Event>? onEvent;
+        public event Action<TransactionUpdate>? onEvent;
+        public event Action<OneOffQueryResponse>? onResult;
```

Comment:
I have mixed feelings about it. Because it's a breaking change either way.

The reason it's TransactionUpdate now is because it's the closest match to the previous Event. But then passing ServerMessage to the SDK users means they get to dispatch on its enum variant.

If that's fine, happy to make the change.

Reply:
Let's go with this:

```C#
public event Action<ServerMessage>? onEvent;
```